docs/advanced_topics/customisation/custom_bulk_actions.rst

```diff
@@ -0,0 +1,123 @@
+Adding custom bulk actions
+==========================================
+
+This document describes how to add custom bulk actions to different listings
+
+
+Registering a custom bulk action
+--------------------------------
+
+    .. code-block:: python
```

Comment:
Note: this misses a few tricks such as how to pass additional context, how to do permission checks and customise the success message.

Reply:
I was wondering if I should add them. Will do :thumbsup:

Reply:
#7425 is a good example of documentation of a new feature that spans across models/views/templates